mcstatus/bedrock_status.py

```diff
@@ -11,7 +11,10 @@
 
 
 class BedrockServerStatus:
-    request_status_data = b"\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\x00\xfe\xfe\xfe\xfe\xfd\xfd\xfd\xfd\x124Vx"
+    request_status_data = bytes.fromhex(
+        # see https://wiki.vg/Raknet_Protocol#Unconnected_Ping
+        "01" + "000000000000000000" + "ffff00fefefefefdfdfdfd12345678" + "a20871356d207e61"
```

Comment:
Black change terrible here, let's skip it.
```suggestion
        "01" + "000000000000000000" + "ffff00fefefefefdfdfdfd12345678" + "a20871356d207e61"  # fmt: skip
```